_posts/en/newsletters/2019-05-07-newsletter.md

```diff
@@ -0,0 +1,273 @@
+---
+title: 'Bitcoin Optech Newsletter #45'
+permalink: /en/newsletters/2019/05/07/
+name: 2019-05-07-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter highlights some of the changes in the
+recently-released Bitcoin Core 0.18.0, describes how bech32 addresses
+are forward compatible with expected protocol improvements, and
+summarizes notable changes in popular Bitcoin infrastructure projects.
+
+## Action items
+
+- **Consider upgrading to Bitcoin Core 0.18.0:** released last week, the
+  latest version of Bitcoin Core brings new features, performance
+  improvements, and bug fixes.  See the *news* section below for
+  more details.
+
+## News
+
+- **Bitcoin Core 0.18.0 released:** this new major version contains
+  several significant new features plus many minor features and
+  bugfixes.  The project's [release notes][0.18 notes] contains a description of
+  all notable changes and a list of each of the 119 people whose merged
+  PRs helped contribute to this release.  Some features we think readers
+  of this newsletter might find especially interesting include,
+
+    - *More PSBT tools and refinements:* the previous major release,
+      0.17, introduced support for [BIP174][] Partially-Signed Bitcoin
+      Transactions (PSBTs) designed to help multiple programs or devices
+      collaboratively create and sign transactions, such as multisig
+      wallets, hardware wallets, and cold wallets.  The 0.18 release
+      builds on that foundation with several bugfixes and improvements,
+      including three new RPCs: `joinpsbts` to merge together multiple
+      PSBTs; `analyzepsbt` to tell the user what they next need to do
+      with the PSBT; and `utxoupdatepsbt` to add necessary information
+      to a PSBT from a node's UTXO set.  Additionally, new [PSBT
+      documentation][] is provided---it's based on the PSBT section from
```

Comment:
nit: 'provided---it's based on...' seems a bit disjointed. How about "provided, which is based on ..."?

Reply:
https://github.com/bitcoinops/bitcoinops.github.io/pull/139#discussion_r281443049